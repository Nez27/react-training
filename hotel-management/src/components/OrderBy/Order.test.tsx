import renderer from 'react-test-renderer';

import Order from '.';
import { BrowserRouter } from 'react-router-dom';

describe('<Order.test />', () => {
  const options = [
    {
      value: 'Value 1',
      label: 'Option 1',
    },
    {
      value: 'Value 2',
      label: 'Option 2',
    },
  ];

  const wrapper = renderer.create(
    <BrowserRouter>
      <Order options={options} />
    </BrowserRouter>
  );

  test('render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
