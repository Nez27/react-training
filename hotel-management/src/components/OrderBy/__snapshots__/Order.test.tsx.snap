// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Order Should render correctly 1`] = `
<div
  className="sc-bdnyFh aZFGz"
>
  <button
    active={true}
    className="sc-gtsqUy fohQqJ"
    disabled={true}
    onClick={[Function]}
  >
    Option 1
  </button>
  <button
    active={false}
    className="sc-gtsqUy bwnpRq"
    disabled={false}
    onClick={[Function]}
  >
    Option 2
  </button>
</div>
`;
